src/multidim/intervalbox.jl

```diff
@@ -0,0 +1,55 @@
+# This file is part of the ValidatedNumerics.jl package; MIT licensed
```

Comment:
I think this file should be in IntervalArithmetic.jl; see [this comment](https://github.com/JuliaIntervals/IntervalRootFinding.jl/pull/8#issuecomment-299942802)...